tests/search.test.ts

```diff
@@ -1435,10 +1437,66 @@ describe.each([
     try {
       await client.health();
     } catch (e: any) {
-      expect(e.cause.message).toEqual("Error: Request Timed Out");
+      expect(e.cause.message).toEqual("request timed out after 1ms");
       expect(e.name).toEqual("MeiliSearchRequestError");
     }
   });
+
+  test(`${permission} key: search should be aborted on already abort signal`, async () => {
```

Comment:
I don't understand the name of this test. Can you explain, or rephrase it, please?

Reply:
First it tests whether search gets immediately aborted on an already aborted signal, then it test whether it gets aborted after the request has already been sent. I have renamed it to `${permission} key: search should be aborted on abort signal`.